### PR TITLE
chore: Add code of conduct, contributing, security and support docs.

### PR DESCRIPTION
**Pull request by ajmirsky**
Based on examples from other open-source repos in github:

- code of conduct for those participating in the project
- instructions for contributing to the project (including a pull request template to help facilitate)
- how to report a security vulnerability
- expectations of getting help with the project

Contributes to https://github.com/mobilityhouse/ocpp/issues/671

### CODE REVIEW

**Comment by jainmohit2001** [comment]
Thanks for the PR! You have covered almost everything.
Another request. Can you please share the reference/source if you have used one?
For COC, its already there, but for the reset, I think we can mention the reference in the PR (as a comment, NOT as a change via commit).

**Comment by ajmirsky** [comment]
contributing was inspired by: https://github.com/probot/template/blob/master/CONTRIBUTING.md

security was inspired by: 
- https://github.com/microsoft/.github/blob/main/SECURITY.md
- https://github.com/skills/.github/blob/main/SECURITY.md
- https://github.com/standard/.github/blob/master/SECURITY.md?plain=1

support from:
- https://github.com/remarkjs/.github/blob/main/support.md?plain=1

pull request based on a few of the examples here:
- https://axolo.co/blog/p/part-3-github-pull-request-template

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Supported Versions
+
+The currently supported versions of this OCPP project are:
+
+| Version  | Supported          |
+|----------| ------------------ |
+| 2.0.0    | :white_check_mark: |
+| 0.26.0   | :white_check_mark: |
+| < 0.26.0 | :x:                |
+
```
Comment (ajmirsky) [inline SECURITY.md:14]: @jerome-benoit  actually, maybe there has been a release? I see releases listed in the [changelog](https://github.com/mobilityhouse/ocpp/blob/master/CHANGELOG.md) and there are tags listed, although none of them are listed in github as a release. and pypi has ocpp 1.0.

Comment (ajmirsky) [reply SECURITY.md:14]: but [github's releases](https://github.com/mobilityhouse/ocpp/releases) only have 0.26.0 as the last official release, although there is a 1.0.0-rc?

Comment (jerome-benoit) [reply SECURITY.md:14]: Given the current state of affair, an automated release process syncing deterministically future versions between GitHub and pypi has higher priority than the others comment I've done on that file in the other PR.

Comment (ajmirsky) [reply SECURITY.md:14]: @jerome-benoit just submitted a new PR #688 which changes the release workflow process to use the same version in pyproject.toml to publish to pypi, create a git tag and create a github release. Let me know if this addresses your concern?

**Review by jainmohit2001 — DISMISSED** [review]

**Comment by jainmohit2001** [comment]
LGTM

**Comment by jainmohit2001** [comment]
@proelke @mdwcrft @jerome-benoit, do we need to make any changes in this PR?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Work in Progress pull requests are also welcome to get feedback early on, or if there is something blocked you.
+
+## Submitting a pull request
+
+1. [Fork][fork] and clone the repository.
+1. Create a new branch: `git checkout -b my-branch-name`.
+1. Configure and install the dependencies: `poetry install`.
+1. Make sure the tests pass on your machine: `make install & make tests`
+1. Make your change, add tests, and make sure the tests still pass.
+1. Push to your fork and [submit a pull request][pr] and complete the information in the pull request template.
```
Comment (mdwcrft) [inline CONTRIBUTING.md:27]:
```suggestion
2. Create a new branch: `git checkout -b my-branch-name`.
3. Configure and install the dependencies: `poetry install`.
4. Make sure the tests pass on your machine: `make install & make tests`
5. Make your change, add tests, and make sure the tests still pass.
6. Push to your fork and [submit a pull request][pr] and complete the information in the pull request template.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Linting requirements
+
+using `make install & make tests` will also run the following linters:
+
+- [Black: The uncompromising Python code formatter](https://black.readthedocs.io/en/stable/)
+- [isort your imports, so you don't have to](https://pycqa.github.io/isort/)
+- [flake8: Your Tool For Style Guide Enforcement](https://flake8.pycqa.org/en/latest/)
+
```
Comment (mdwcrft) [inline CONTRIBUTING.md:36]:
```suggestion

using `make format` will run isort and black and apply any formatting suggestions from them

```

**Review by jainmohit2001 — APPROVED** [review]

**Pull request merged**